src/App.tsx

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useEffect,
+  useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { User } from './types/User';
+
+function getFilteredTodos(
+  todos: Todo[],
+  query: string,
+  sortBy: string,
+) {
+  enum Status {
```

Comment:
Let's move enum to `types` folder in a separate file and use it across the app for defining filter status (in state, etc.)